### ISSUE TITLE
no henchmen or bosses

### ISSUE THREAD

**Issue by aleccq** [RealProHenis/ButterflyLauncher#34]
version 12.21 works great but no henchmen or bosses, any fix?

**Comment by Drixxytec** [comment]
Yeah, uhh. You can't have that, that's normal because then he'd have to code A.I. on the gameserver and nobody's gonna really do that unless you use another gameserver but still no Henchmen or Bosses. Sorry but those are impossible.
